.gitignore

```diff
@@ -1,3 +1,7 @@
+# IDEs
```

Comment:
let's use this instead:
```
# intellij configs
```

(same as done in the other project)

Reply:
✅